cloud/google/mirror-config.sh

```diff
@@ -31,16 +34,18 @@ for i in $(seq ${MIRROR_NUM_REPLICAS}); do
 done
 
 
-export PROMETHEUS_NUM_REPLICAS=1
-export PROMETHEUS_DISK_SIZE=50GB
-export PROMETHEUS_BASE_NAME="${CLUSTER}-prometheus"
-export PROMETHEUS_MACHINE_TYPE=n1-standard-1
-declare -a PROMETHEUS_MACHINES MIRROR_DISKS
-export PROMETHEUS_MACHINES MIRROR_DISKS
-for i in $(seq ${PROMETHEUS_NUM_REPLICAS}); do
-  PROMETHEUS_MACHINES[$i]="${CLUSTER}-prometheus-$i"
-  PROMETHEUS_DISKS[$i]="${CLUSTER}-prometheus-disk-$i"
-done
+if [ "${MONITORING}" == "prometheus" ]; then
```

Comment:
Yeah, there's definitely some things which can be pulled out, but I'll leave that for a follow-up if that's ok.